challenge2/styles.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    /* Colors */
+    --primary-color: #FCFCFC;
+    --complimentary-red: #EF4444;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
+    font-family: "Ubuntu", sans-serif;
+}
+
+header p {
+    font-size: 1.8rem;
+}
+
+.main-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 20px;
+    padding-bottom: 25%;
```

Comment:
Are these paddings needed? Particularly, the 25% seems a bit weird, I don't see that in the Figma design